### DISCUSSION TITLE
moved zigbee2mqtt from Raspberry to Linux Ubuntu. server starts but webserver on port 8080 doesn't work

### DISCUSSION THREAD

**Discussion by laurenttaieb** [Koenkk/zigbee2mqtt#17071] [General]
Hi Forum,
I moved zigbee2mqtt from my Raspberry 3 to a Linux Ubuntu on minipc.
I followed the procedure defined on https://www.zigbee2mqtt.io/ 
Server looks to launch properly and looking at the log on /data, it shows that the coordinator has been found as well as a number of devices
I also have the message 
info  2023-03-18 16:19:03: Connecting to MQTT server at mqtt://localhost:1883
info  2023-03-18 16:19:03: Connected to MQTT server

when I query the daemon, I'm getting the following response:

     Loaded: loaded (/etc/systemd/system/zigbee2mqtt.service; enabled; vendor preset: enabled)
     Active: active (running) since Sat 2023-03-18 16:18:50 CET; 19min ago
   Main PID: 1422339 (npm start)
      Tasks: 23 (limit: 9308)
     Memory: 93.7M
        CPU: 3.226s
     CGroup: /system.slice/zigbee2mqtt.service
             ├─1422339 "npm start" "" "" "" "" "" "" "" "" "" "" "" "" "" ""
             ├─1422350 sh -c "node index.js"
             └─1422351 node index.js

Home Assistant is showing a number of devices from the Zigbee2mqtt integration but I cannot query the web server on port 8080 and all devices are showing on HA as offline.
Can someone provide some help ? I've looked at multiple options but none seems to work.
Thanks
Laurent

**Comment by elronzo** [comment]
Firewall aktiv? Möglicherweise richtets ein `sudo ufw allow 8080/tcp`.

**Comment by laurenttaieb** [comment]
I tried . It didn’t work.I checked and 8080 is not open.Thanks Laurent Le 19 mars 2023 à 12:22, elronzo ***@***.***> a écrit :﻿
Firewall aktiv? Möglicherweise richtets ein sudo ufw allow 8080/tcp.
